l10n_br_pos/models/pos_config.py

```diff
@@ -0,0 +1,195 @@
+# © 2016 KMEE INFORMATICA LTDA (https://kmee.com.br)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+PRINTER = [
+    ("epson-tm-t20", "Epson TM-T20"),
+    ("bematech-mp4200th", "Bematech MP4200TH"),
+    ("daruma-dr700", "Daruma DR700"),
+    ("elgin-i9", "Elgin I9"),
+]
+
+SIMPLIFIED_INVOICE_TYPE = [
+    ("nfce", "NFC-E"),
+    ("sat", "SAT"),
+    ("paf", "PAF-ECF"),
+]
+
+
+class PosConfig(models.Model):
+    _inherit = "pos.config"
+
+    @api.depends("out_pos_fiscal_operation_id")
+    def _compute_allowed_tax(self):
+        for record in self:
+            record.cfop_ids = record.cfop_ids.search([("is_pos", "=", True)])
+            if record.cfop_ids and record.out_pos_fiscal_operation_id:
+                record.out_pos_fiscal_operation_line_ids = (
+                    record.out_pos_fiscal_operation_id.line_ids.filtered(
+                        lambda x: x.cfop_internal_id in record.cfop_ids
+                    )
+                )
+            else:
+                record.out_pos_fiscal_operation_line_ids = False
+
+    out_pos_fiscal_operation_id = fields.Many2one(
+        comodel_name="l10n_br_fiscal.operation",
+        string="Operação Padrão de Venda",
+        default=lambda self: self.env.company.out_pos_fiscal_operation_id,
+    )
+
+    partner_id = fields.Many2one(
+        comodel_name="res.partner",
+    )
+    # TODO: Isso pode ser uma one2many
+
+    cfop_ids = fields.One2many(
+        string="CFOPs permitidas",
+        comodel_name="l10n_br_fiscal.cfop",
+        compute="_compute_allowed_tax",
+        readonly=True,
+    )
+
+    out_pos_fiscal_operation_line_ids = fields.One2many(
+        comodel_name="l10n_br_fiscal.operation.line",
+        string="Linhas de Operação de Venda",
+        compute="_compute_allowed_tax",
+        readonly=True,
+    )
+
+    refund_pos_fiscal_operation_id = fields.Many2one(
+        comodel_name="l10n_br_fiscal.operation",
+        string="Operação Padrão de Devolução",
```

Comment:
Pessoal.. eu recomendo mudar as mensagens e titulos tudo para inglês. Estamos tratando os casos anteriores e acho que seria valido já entrar este módulo com tudo em ingles.

Reply:
Eu acho super relevante pros módulos centrais para as empresas multinacionais, como base, account, fiscal, sale, purchase... Ja para módulos mais especializados, seria legal mas eu acho menos critico (primeiro porque ja é complicado/raro chegar ate esses modulos, segundo porque mesmo se uma multinacional chega até esses módulos, quem for botar a mão vai ter que dominar a lingua de qualquer jeito). Resumindo: eu acharia legal, mas tb não faço questão de blocar por isso nessa altura.

Reply:
Entendo. Neste caso concordo que nao seja algo critico mas seria legal para facilitar a gestão das traduções.

Reply:
> Entendo. Neste caso concordo que nao seja algo critico mas seria legal para facilitar a gestão das traduções já que se vc nao traduzir fica constando a falta de tradução (nada demais) e se vc for traduzir igual ao termo original as ferramentas reclamam.

interesse para ficar de fica quem for iniciar novos trabalhos aqui na localização

Reply:
@ygcarvalh show.. acho que melhor aproveitar agora para corrigir mesmo.. em algum momento alguem acabaria precisando fazer.. vlwwww